content/examples/databricks-cluster-local.Rmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "Option 2 - Working with Databricks locally"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+If the recommended path of connecting to Spark remotely with Databricks Connect
+does not apply to your use case, then you can install RStudio Server Pro
+directly within a Databricks cluster.
+
+With this configuration, RStudio Server Pro is installed on the Spark driver
+node and allows users to work locally with Spark.
+
+<img src="/images/deployment/databricks/rstudio-databricks-local.png" width='800px' align='center'/>
+
+This configuration can result in more complex configuration, limited
+connectivity to other storage and compute sources, resource contention between
+RStudio Server Pro and Databricks, and maintenance concerns due to the ephemeral
+nature of Databricks clusters.
+
+For additional details, refer to the
+[FAQ for RStudio in the Databricks Documentation](https://docs.databricks.com/spark/latest/sparkr/rstudio.html#frequently-asked-questions-faq).
+
+## Advantages and limitations
+
+Advantages:
+
+- Can load data from the cluster since RStudio Server Pro is installed within
+  the Databricks cluster
```

Comment:
I'm not sure if this is meant as an advantage over connecting remotely. When you connect remotely, you should be able to access data from the cluster as well. Unless "load" in this case means loading data into the R session, not "loading" the data into Spark. Either way, I think some clarification is needed here.

Reply:
To clarify, this was referring to loading data directly into memory in the R session. I've seen this performed commonly enough outside of pyspark/sparklyr due to the convenience of being on a cluster node.

I modified the description in 442d919, but I can remove this point if you still think it adds more confusion.